fury/shaders/sdf_impl.frag

```diff
@@ -0,0 +1,37 @@
+/* SDF fragment shader implementation */
+
+//VKT::Light::Impl
+
+vec3 point = centeredVertexMC;
+
+vec3 uu = vec3(MCVCMatrix[0][0], MCVCMatrix[1][0], MCVCMatrix[2][0]); // camera right
+vec3 vv = vec3(MCVCMatrix[0][1], MCVCMatrix[1][1], MCVCMatrix[2][1]); //  camera up
+vec3 ww = vec3(MCVCMatrix[0][2], MCVCMatrix[1][2], MCVCMatrix[2][2]); // camera direction
+
+//ray origin
+vec4 ro = -MCVCMatrix[3] * MCVCMatrix;  // camera position in world space
+
+vec3 col = vertexColorVSOutput.rgb;
+
+//ray direction
+vec3 rd = normalize(point - ro.xyz);
```

Comment:
I think the deformation problem come from here.
- `point` = `centeredVertexMC` = Model space
- `ro` = `MCVCMatrix` =  View / World space
I think we should look deeper here.